ROADMAP.md

```diff
@@ -8,17 +8,10 @@ core team member that nobody else is working on the same thing.
 **Note:** This doesn’t include everything that the core team will work
 on, and everything is subject to change.
 
-- Make it possible to run mypy as a daemon to avoid reprocessing the
-  entire program on each run. This will improve performance
-  significantly. Even when using the incremental mode, processing a
-  large number of files is not cheap.
-
 - Provide much faster, reliable interactive feedback through
```

Comment:
This is also outdated I would say. Fine grained mode (used by daemon) is of solid beta quality now.

Reply:
Yes, this can be removed as well.